src/app/access-requests/features/access-grant-revocation-dialog/access-grant-revocation-dialog.html

```diff
@@ -1,31 +1,42 @@
-<div class="w-full sm:p-8">
+<div class="w-full">
   <h2 mat-dialog-title>
     <span class="fa-file-download"></span>Access Grant Revocation
   </h2>
   <mat-dialog-content>
-    <p>Type the Grant ID ({{ grantID }}) and click 'Revoke' to revoke the grant.</p>
+    <p>
+      Please confirm that the access grant of
+      <strong
+        >{{ grant.user_title ? grant.user_title + ' ' : '' }}{{grant.user_name}}</strong
+      >
+      with email <strong>{{ grant.user_email }}</strong> with ID
+      <strong>{{ grant.id}}</strong> shall be <strong>revoked</strong> by writing the
+      grant ID in the box below.
+    </p>
     <mat-form-field class="w-full">
       <input
-        [(ngModel)]="idInput"
+        [(ngModel)]="userInput"
         matInput
         class="w-full"
-        placeholder="Type the Grant ID here"
+        placeholder="Confirm grant ID"
```

Comment:
I think we should not use the grant ID here. It's a long UUID that is not relevant to the data steward, does not carry any information, and requesting it would be just an exercise in copy & paste.

Instead, we should show and request the dataset accession (much shorter) _and_ the user name. When validating the user name, we should ignore the title (i.e. allow input with or without title). Typing or copying the dataset ID and name will help the data steward to double check it's really the one that shall be deleted.